Cargo.toml

```diff
@@ -20,15 +20,16 @@ is-it-maintained-open-issues = { repository = "cardoe/stderrlog-rs" }
 maintenance = { status = "passively-maintained" }
 
 [dependencies]
-atty = "^0.2, <=0.2.11"
-chrono = "0.4"
+atty = "0.2"
+chrono = "0.4.1"
 log = { version = "0.4.1", features = ["std"] }
-termcolor = "1.0"
-thread_local = "^0.3, <=0.3.4"
+termcolor = "~1.1"
+thread_local = "~1.0"
 
 [dev-dependencies]
-clap = "~2.22.0"
-docopt = "0.6"
-rustc-serialize = "0.3"
-libc = "0.2"
-structopt = "0.2"
+clap = "~2.33.0"
+docopt = "1.1.0"
+serde = "1.0"
+serde_derive = "1.0"
```

Comment:
You got rid of the extern crates but then didn't import serde appropriately.
`serde = { version = "1.0", features = ["derive"] }` and getting rid of `serde_derive`

Reply:
Looks like you fixed that in your version, thanks.